src/cdk/table/table.ts

```diff
@@ -666,7 +673,20 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
     });
 
     this._updateNoDataRow();
-    this.updateStickyColumnStyles();
+
+    // Allow the new row data to render before measuring it.
+    // TODO: Remove indirection once _ngZone is required.
```

Comment:
This should have a `@breaking-change` tag instead of a `TODO` so that our tooling can pick it up.